GameOfThrones/Migrations/Configuration.cs

```diff
@@ -0,0 +1,101 @@
+﻿namespace GameOfThrones.Migrations
+{
+    using GameOfThrones.Models;
+    using System;
+    using System.Data.Entity;
+    using System.Data.Entity.Migrations;
+    using System.Linq;
+
+    internal sealed class Configuration : DbMigrationsConfiguration<GameOfThrones.DAL.WikiContext>
+    {
+        public Configuration()
+        {
+            AutomaticMigrationsEnabled = false;
+        }
+
+        protected override void Seed(GameOfThrones.DAL.WikiContext context)
+        {
+            context.Castles.AddOrUpdate(i => i.Name,
+                new Castle
+                {
+                    Name = "The Eyrie"
+                },
```

Comment:
When I try to generate the database locally I'm getting
```
Applying explicit migrations: [202109141059199_AddCharacter].
Applying explicit migration: 202109141059199_AddCharacter.
System.Data.SqlClient.SqlException (0x80131904): The ALTER TABLE statement conflicted with the FOREIGN KEY constraint "FK_dbo.Characters_dbo.Houses_HouseId_HouseId". The conflict occurred in database "GameOfThrones.DAL.WikiContext", table "dbo.Houses", column 'HouseId'.
```
You would still need to set the ID in the seed for House/Castle, I think this is why I'm seeing those errors when I'm trying to generate a fresh database